CHANGES.rst

```diff
@@ -8,6 +8,16 @@
 Changes
 =======
 
+Version 1.0.1 (released 2018-06-13)
```

Comment:
@lnielsen Is this fully release-ready? We should probably update the date to tomorrow then...  Can take care of releasing afterwards.